packages/amplify-graphql-index-transformer/src/resolvers/resolvers.ts

```diff
@@ -418,25 +427,28 @@ export function updateResolversForIndex(
   const deleteResolver = getResolverObject(config, ctx, 'delete');
   const syncResolver = getResolverObject(config, ctx, 'sync');
 
+  const dbType = getDBType(ctx, object.name.value);
+  const isDynamoDB = dbType === 'DDB';
+
   // Ensure any composite sort key values and validate update operations to
   // protect the integrity of composite sort keys.
-  if (createResolver) {
+  if (isDynamoDB && createResolver) {
```

Comment:
Is this method common to both DDB and RDS? If not, I would prefer we move it to DDB VTL generator as well. That would avoid such conditional checks.

Reply:
Yes, it is common for RDS and DDB.